packages/vue-components/src/Searchbar.vue

```diff
@@ -179,7 +180,8 @@ export default {
         'dropdown-menu',
         'search-dropdown-menu',
         { show: this.showDropdown },
-        { 'dropdown-menu-right': this.menuAlignRight },
+        { 'd-none': !this.showDropdown },
+        { 'dropdown-menu-end': this.menuAlignRight },
```

Comment:
The dropdown for the searchbar is also a little buggy. The first time is fine, aligned to the right but on the second time onward, the dropdown is aligned to the left.

https://user-images.githubusercontent.com/47470981/162556182-e312094f-7e50-42df-a51b-c542f4618955.mp4